packages/node/src/handlers.ts

```diff
@@ -292,9 +293,15 @@ export function errorHandler(): (
       next(error);
       return;
     }
-    const eventId = captureException(error);
-    (_res as any).sentry = eventId;
-    next(error);
+    withScope(scope => {
+      if (isString(_req.headers['sentry-trace'])) {
```

Comment:
Is this given that `headers` will always be at least an empty object? otherwise it will throw if `headers === undefined`